July-Morning/assignment4/disjoint_set.hpp

```diff
@@ -0,0 +1,21 @@
+#ifndef DISJOINT_SET_HPP_
+#define DISJOINT_SET__HPP_
+
+#include <vector>
+
+class DisjointSet
```

Comment:
Very good, DisjointSet class which is portable and is not coupled with this particular solution. :+1: